tests/Tests/500__jupyterhub/pvc-change_verification.robot

```diff
@@ -0,0 +1,125 @@
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource        ../../Resources/Page/ODH/JupyterHub/ODHJupyterhub.resource
+Suite Teardown  PVC Size Suite Teadrown
+Test Setup      PVC Size Test Setup
+
+*** Variables ***
+${NAMESPACE}    redhat-ods-applications
+${S_SIZE}       4
+${SIZE_CODE}    import subprocess;
+...    int(subprocess.check_output(['df','-h', '/opt/app-root/src']).split()[8].decode('utf-8')[:-1])
+@{NS_SIZE}      0    abc    6.5   -12
+
+
+*** Test Cases ***
+Verify User Can Spawn Notebook After Changing PVC Size Using Backend
+    [Documentation]   Verify if user can spawn notebook
+    ...    for supported PVC size got changed
+    [Tags]    Smoke
+    ...       Sanity
+    ...       Tier1
+    ...       ODS-1221
+    Change And Apply PVC size    ${S_SIZE}Gi
+    Run Keyword And Warn On Failure   Verify Notebook Size     600s    ${S_SIZE}
+    ${pvc_size}   Get Notebook PVC Size        username=${TEST_USER.USERNAME}   namespace=rhods-notebooks
+    Verify PVC Size     ${S_SIZE}       ${pvc_size}
+    [Teardown]    PVC Size Test Teardown
+
+Verify User Cannot Set An Unsupported PVC Size Using Backend
+    [Documentation]   Verify if user should not able to
+    ...    spawn notebook for supported PVC change
+    [Tags]    Tier2
+    ...       ODS-1229
+    Verify Multiple Unsupported Size    ${NS_SIZE}
+    [Teardown]    PVC Size Test Teardown
+
+Verify User Can Spawn Notebook After Changing PVC Size Using UI
+    [Documentation]   Verify if dedicated admin user able to chnage PVC
+    ...    and RHODS user is able to spawn notebook for supported PVC
+    ...    and verify PVC size
+    [Tags]    Smoke
+    ...       Sanity
+    ...       Tier1
+    ...       ODS-1220    ODS-1222
+    Verify PVC change using UI     ${S_SIZE}
+    ${pvc_size}   Get Notebook PVC Size        username=${TEST_USER.USERNAME}   namespace=rhods-notebooks
+    Verify PVC Size     ${S_SIZE}       ${pvc_size}
+    [Teardown]    PVC Size UI Test Teardown
+
+Verify User Cannot Set An Unsupported PVC Size Using The UI
+        [Documentation]   Verify if dedicated admin user able to chnage PVC
+    ...    and RHODS user is able to spawn notebook for unsupported PVC
+    ...    and verify PVC size
+    [Tags]    Tier2
+    ...       ODS-1223
+    FOR    ${size}    IN    @{NS_SIZE}
+         Verify PVC change using UI   ${size}
+         ${pvc_size}   Get Notebook PVC Size        username=${TEST_USER.USERNAME}   namespace=rhods-notebooks
+         ${status}    Run Keyword And Return Status    Verify PVC Size     ${size}       ${pvc_size}
+         IF   '${status}' != 'True'
+               Log     Actul size and assigned size is mismatch
+         ELSE
+               Log     User is able to spawn and set the Unsupported size
+               Run Keyword And Continue On Failure    Fail
+         END
+         PVC Size Suite Teadrown
+    END
+    [Teardown]    PVC Size UI Test Teardown
+
+*** Keywords ***
+Launch RHODS Dashboard
```

Comment:
this keyword is already implemented in `Resources/Page/ODH/ODHDashboard.robot`, pls re-use it